astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -386,7 +386,7 @@ class Module(LocalsDictNodeNG):
     <Module l.0 at 0x7f23b2e4eda0>
     """
 
-    _astroid_fields = ("body",)
+    _astroid_fields = ("body", "doc_node")
```

Comment:
@cdce8p I just caught something.

https://github.com/PyCQA/astroid/blob/841401d48be94c1bec179864f36ac11beea61f97/astroid/nodes/node_ng.py#L254-L263

`last_child` iterates over this with `-1`. So I think the order of `astroid_fields` matters. I haven't looked at an actual edge case yet, but we might need to change this around?
```python
def func():
    """Test"""
    a = 1
```
I think the docstring will now be the last child in this example?

Reply:
Good catch!